operator/api/v1alpha1/internalworkspace_types.go

```diff
@@ -34,8 +36,13 @@ const (
 	// LabelInternalDomain domain for internal labels
 	LabelInternalDomain string = "internal.workspaces.konflux.io/"
 
-	// PublicViewerName the name of the KubeSaw's PublicViewer user
-	PublicViewerName string = "kubesaw-authenticated"
+	// ConditionTypeReady indicates whether an InternalWorkspace is Ready
+	ConditionTypeReady string = "Ready"
+	// ConditionReasonEverythingFine indicates "everything is fine"
+	ConditionReasonEverythingFine string = "EverythingFine"
+	// ConditionReasonOwnerNotFound means that the UserSignup for the InternalWorkspace
+	// was not found
+	ConditionReasonOwnerNotFound string = "OwnerNotFound"
```

Comment:
Only two reasons one for OK/Fine and the other very specific about owner not found, what if there is an error should we distinguish between owner not found and an error making the calls?

Reply:
in this case the reconciler won't update the status but it will requeue and try again. For now, I'd say we don't need a condition for errors making calls